operations/kubernetes/Backups_for_Etcd_Clusters_Running_in_Kubernetes.md

```diff
@@ -13,7 +16,71 @@ The following services are backed up daily \(one week of backups retained\) as p
 - Boot Orchestration Service \(BOS\)
 - Boot Script Service \(BSS\)
 - Firmware Action Service \(FAS\)
-- User Access Service \(UAS\) - Deprecated
+- Heartbeat Tracking Daemon \(HBTD\)
+- HMS Notification Fanout Daemon \(HMNFD\)
+- Power Control Service \(PCS\)
+- User Access Service \(UAS\)
+
+## Clusters without automated backups
+
+The following projects are not backed up as part of the automated solution:
+
+- Content Projection Service \(CPS\)
+
+If these clusters become unhealthy, the process for rediscovering their data should be followed.
+See [Repopulate Data in etcd Clusters When Rebuilding Them](Repopulate_Data_in_etcd_Clusters_When_Rebuilding_Them.md).
+
+## Test for recent etcd cluster backups
+
+This test will show PASS or FAIL for each of the etcd clusters as it verifies whether they had a backup in the last 24 hours.
+
+(`ncn-mw#`) To view all available etcd backups across all clusters
+
+```bash
+/opt/cray/platform-utils/ncnHealthChecks.sh -s etcd_backups_check
+```
+
+Example output:
+
+```text
+**************************************************************************
+
+=== Verify etcd clusters have a backup in the last 24 hours. ===
+=== The complete list of backups can be listed as follows:
+=== % /opt/cray/platform-utils/etcd/etcd-util.sh list_backups -
+Tue 08 Oct 2024 05:37:02 PM UTC
+
+-- cray-bos -- backups
+PASS: backup found less than 24 hours old.
+
+-- cray-bss -- backups
+PASS: backup found less than 24 hours old.
+
+-- cray-fas -- backups
+PASS: backup found less than 24 hours old.
+
+-- cray-fox -- backups
```

Comment:
I don't see fox listed in the bulleted list further up this page

Reply:
It was hard to find the description of what FOX means which is why I originally left it out.  It is not an acronym.  It was developed by HMS team and now is part of the CSM Diags team. 

- Controller Diagnostics Orchestration (FOX)

I found this description at 
https://rndwiki-pro.its.hpecorp.net/display/CASMHMS/Fox+-+Controller+Diagnostics+Orchestration+API

Fixed.